### DISCUSSION TITLE
Grafana dashboard provisioning. Add a dashboard json name in logs  warning message

### DISCUSSION THREAD

**Discussion by lutskevich** [grafana/grafana#71565] [General]
Please, add a dashboard name or json title (and folder) of duplicated dasboard in logs message
"message": ["logger=provisioning.dashboard t=2023-07-13T13:18:19.776231564Z level=warn msg=\"the same UID is used more than once\" orgId=1 uid=61e87a5 times=2 providers=[sidecarProvider]"]
it can be difficult to find a dashboard by uid, there are a lot of them in different projects, there is no access

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82607. Feel free to continue the discussion around this there. Thank you!
